src/components/Table/nestedCards.tsx

```diff
@@ -214,6 +214,27 @@ export function getNestedCardStyles(
   };
 }
 
+/**
+ * Returns the background color for the row.
+ * Either based on the current row's card having a defined background color, or the last opened card's background color
+ * Defaults to White.
+ */
+export function getCurrentBgColor(
+  row: GridDataRow<any>,
+  openCardStyles: NestedCardStyle[] | undefined,
+  nestedKinds: NestedCardStyleByKind,
+): string {
+  if (nestedKinds[row.kind]) {
```

Comment:
Huh, is it possible to use nested cards w/o having a `nestedKind` entry for _every_ kind?

Reply:
Yes it is.